src/lib/viewers/doc/_docBase.scss

```diff
@@ -202,6 +203,18 @@ $thumbnail-sidebar-width: 191px; // Extra pixel to account for sidebar border
 }
 
 @include breakpoint($medium-screen) {
+    .bp-PageControls-button {
+        display: none;
+    }
+
+    .bp-PageControlsForm-button-label {
+        min-width: $bp-controls-size-large;
+    }
+
+    input[type='number'].bp-PageControlsForm-input {
```

Comment:
We should try to avoid tag-based styling whenever possible. Can we use a class name?